doctests/dt-json.js

```diff
@@ -0,0 +1,382 @@
+// EXAMPLE: json_tutorial
+// HIDE_START
+import assert from 'assert';
+import {
+  createClient
+} from 'redis';
+
+const client = await createClient();
```

Comment:
pretty sure this wont add json to the client. should need to specify it as a module on the client.